CMakeLists.txt

```diff
@@ -371,7 +371,15 @@ endif()
 option(CPPINTEROP_INCLUDE_DOCS "Generate build targets for the CppInterOp docs.")
 option(CPPINTEROP_ENABLE_DOXYGEN "Use doxygen to generate CppInterOp interal API documentation.")
 option(CPPINTEROP_ENABLE_SPHINX "Use sphinx to generage CppInterOp user documentation")
-option(CPPINTEROP_ENABLE_TESTING "Enables the testing infrastructure." ON)
+
+
+if(EMSCRIPTEN)
+    message("Build with emscripten")
+    option(CPPINTEROP_ENABLE_TESTING "Enables the testing infrastructure." OFF)
```

Comment:
Any concrete reason, we're not planning to enable testing here ?

Reply:
This was done for no other reason than that was what was done in xeus-cpp (i.e. testing turned off by default when emscripten is on) https://github.com/compiler-research/xeus-cpp/blob/216dbed46e358d82fd8fe3065b4533f3e1c49119/CMakeLists.txt#L77C1-L77C34 .

Reply:
Works can skip that for now !